### PR TITLE
fix(release): updates the release script to prevent git conflicts

### PR DESCRIPTION
**Pull request by robertmesserle**
@ThomasBurleson @jelbourn

### CODE REVIEW

File: release.js [hunk 1]
```diff
@@ -19,11 +19,11 @@
   var dryRun;
 
   header();
-  write('Is this a dry-run? {{"[yes/no]".cyan}} ');
+  write(`Is this a dry-run? ${"[yes/no]".cyan} `);
```
Comment (ThomasBurleson) [inline release.js]: This uses the support for template strings in NodeJS 4.2.x ?

**Comment by ThomasBurleson** [comment]
@robertmesserle - are these changes required due to the upgrade to NodeJS 4.2.x?

**Comment by robertmesserle** [comment]
No they are unrelated.

File: release.js (hunk 1 above)
Comment (robertmesserle) [inline release.js]: Yes, it uses the v4 features in place of the manual method I used before.

**Pull request closed without merging**